### PR TITLE
preserve_ini_comments

### LINKED ISSUE #26: Comments in ini files
I have realized that `pretty-format-ini` removes comment lines. Is this on purpose?

### PR DESCRIPTION
**Pull request by Skylion007**
Closes #26 

I ended up using INIParse since it's more compatible with the ConfigParser library than ConfigObj and more lightweight. Adds one tiny dependency.

### CODE REVIEW

File: language_formatters_pre_commit_hooks/pretty_format_ini.py [hunk 1]
```diff
@@ -36,18 +37,17 @@ def pretty_format_ini(argv=None):
...
+            # INIConfig only supports strict mode for throwing errors
+            config_parser = ConfigParser()
             if PY3:  # pragma: no cover # py3+ only
                 config_parser.read_string(string_content)
             else:  # pragma: no cover # py27 only
                 config_parser.readfp(StringIO(str(string_content)))
-
-            pretty_content = StringIO()
-            config_parser.write(pretty_content)
+            config_parser = INIConfig(StringIO(str(string_content)), parse_exc=False)
```
Comment (cortinico) [inline language_formatters_pre_commit_hooks/pretty_format_ini.py:47]: I'd rather use `ini_config` here instead of `config_parser`

**Review by cortinico — APPROVED** [review]
👍 waiting for @macisamuele to take a look

**Comment by Skylion007** [comment]
Bump @macisamuele

**Comment by macisamuele** [comment]
Sorry missed this ;(
Changes looks good to me. Good to get merged.

I'm not sure on when this will be released on a new version.
You can still use this by using the SHA of master (after merging this in) as version in the pre-commit hook.
If you have a different need and need this to be released please let me know and I'll try to do it sooner (instead around the weekend).

**Pull request merged**